personnel.md

```diff
@@ -0,0 +1,216 @@
+# Personnel
+
+## Current
+
+### Secretaries
+
+| Name                                  | Term                    |
+|---------------------------------------|-------------------------|
+| Alessandro Lai ([@AlessandroLai])     | 2017-11-12 - 2018-08-31 |
+| Margret Staples ([@dead_lugosi])      | 2017-11-12 - 2019-05-31 |
+| Michael Cullum ([@michaelcullumuk])   | 2016-01-31 - 2018-01-31 |
+
+Feel free to contact the secretaries at info AT php-fig.org. For more information on the secretaries' role, see the [bylaws](/bylaws).
+
+
+### Core Committee Members
+
+| Name                                  | Term                    |
+|---------------------------------------|-------------------------|
+| **Group 1**                                                     |
```

Comment:
I don't really like grouping them, term dates do this already.

Reply:
Hmm I thought it made more immediate to look names up alphabetically, here are some images of the final result:
![grouped](https://files.torresi.io/cc-grouped.png)
![ungrouped](https://files.torresi.io/cc-ungrouped.png)

Reply:
I don't like the way groups categorise them.

Reply:
I like the groups, but I think it would get complicated if, i.e., someone resigns and the group get fragmented.

Reply:
My problem with grouping is it implies some kind of seniority and also, how would numbers be decided after January elections for example.